src/js/pages/services/DeploymentsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import AlertPanel from '../../components/AlertPanel';
+
+var DeploymentsTab = React.createClass({
```

Comment:
Sorry you're going through this Phil. I think it was helpful for the rest of the team when we were in HH. Maybe it would make sense for us to do an overview with you since you weren't able to make it.

For now would you mind making this an es6 class since this is a new component?